packages/subapp-util/lib/index.js

```diff
@@ -29,6 +30,15 @@ function removeExt(f) {
   return ix > 0 ? f.substring(0, ix) : f;
 }
 
+const validateModuleManifest = manifest => {
+  const entries = _.pick(manifest, ["entry", "serverEntry", "reducers"]);
+  Object.keys(entries).forEach(x => {
+    if (!Path.isAbsolute(entries[x])) {
+      throw new Error(`Error: manifest ${x} requires absolute path`);
```

Comment:
error message is not very clear

Reply:
Updated the error message in latest commit